### ISSUE TITLE
exception "Undefined index" for all sortable fields

### ISSUE THREAD

**Issue by andycaramba** [yajra/laravel-datatables#114]
Hi there.
After fixes in 33420f1adf2a3e8eb4658ee0722d43dd87c4b4ae i have an exception
```
"Undefined index: created_at" at vendor/yajra/laravel-datatables-oracle/src/yajra/Datatables/Engines/CollectionEngine.php line 98
```
for each sortable field (not only `created_at`)

When I manually change from `return with((array) $row)[$column];` back to `return $row[$column];` in https://github.com/yajra/laravel-datatables/blob/master/src/yajra/Datatables/Engines/CollectionEngine.php#L98 all works fine.

**Comment by yajra** [comment]
What version of Laravel please? I encountered this issue yesterday on Laravel 5.1 and that patch fixed it for me. Thanks for reporting.

**Comment by andycaramba** [comment]
There is part of my composer.json
```
"require": {
    "illuminate/support": "5.1.*",
    "yajra/laravel-datatables-oracle": "~5.0"
  },
```

**Comment by andycaramba** [comment]
And this exception appears only if Collection items is Eloquent Model. If Collection items is Array all works fine.

**Comment by yajra** [comment]
@andycaramba, thanks for reporting. My bad for not being able to test collection engine. Was working with eloquent when I pushed the patch. Should be fixed on v5.5.10. Thanks!

**Comment by andycaramba** [comment]
Now all works fine. Thank you for such an incredibly fast fix )
